packages/nodes-base/nodes/Webhook/utils.ts

```diff
@@ -167,3 +176,96 @@ export const checkResponseModeConfiguration = (context: IWebhookFunctions) => {
 		);
 	}
 };
+
+export async function validateWebhookAuthentication(
```

Comment:
BTW, since this so neatly extracted out now, it might be possible to write unit tests for this. If we can add these tests in the same PR, that's be nice, otherwise I can also try to create a PR with just the tests later 🙏🏽